yang/frr-isisd.yang

```diff
@@ -493,6 +493,13 @@ module frr-isisd {
         description
           "Node protection is provided by the alternate.";
       }
+      leaf link-fallback {
+        type boolean;
+        must ". = 'false' or ../enable = 'true'";
+        default false;
```

Comment:
Wouldn't it be better to have link fallback enabled by default? Once you have the SPTs and reverse SPTs of all adjacent neighbors, computing link fallbacks is relatively inexpensive (just one additional SPF run).

Reply:
In all the drafts/RFCs that I remember talk about this, they say that implementations SHOULD allow enabling such a fallback. So, I believe that having to explicitly indicate this in the config is more aligned with those ?
I don't have a strong opinion on this, though. However, as a user, I found it confusing that, when node protection was enabled, some repair paths were using the protected node. Later I realized that this was because the fallback had silently kicked in.

Reply:
Fair enough. Marking as resolved...